### PR TITLE
Big red SW caves (gitlab port)

### PR DESCRIPTION
**Pull request by PhantomEpicness**
## About The Pull Request
Somehow fixed the massive corruptions in this map by reverting and copy pasting changes onto github base. While it was approved on gitlab,  a re review is required since changes may have been lost in the revert.

Big red southwest caves are notoriously underwelming and underdeveloped at best. To make it worse, there is a hive spot slapped there, barely even a screen away from LZ2, which results in an extremely short round if marines land LZ2 with a SW caves hivespot
Furthermore, I added a new area, 'Abandoned mining', which is features a small depot, office and aswell as several small mining structures scattered in the area.
Also features a few goodies for survs who dare to venture here. Notable loot is some "mining explosive" (c4), a few guns and molotov cocktails + an m74 with some baton slugs

This addition also features some unique loot for survivors. Mining explosive is hidden somewhere there. Find it and you can open podlocks!

###The entire added areas. Note the unique terrian patterns compared to the rest of the caves in big red.

Screenshot_356

(eastereggs photoshopped out :D )

## Why It's Good For The Game
reworks a very neglected and underwhelming part of the best map.

## Changelog

:cl: TotalEpicness5
add: Expanded big red's southwestern section.
add: Created 'Abandoned mining caves' an abandoned section of big red SW caves that features new cave networks and a warehouse for tight combat!
add: Moved big red SW caves hive spawn point to Abandoned Mining.
/:cl:

### CODE REVIEW

**Comment by PurpleCIoud** [comment]
I like it

**Comment by MusselMain** [comment]
Finally, Trijent and Solaris never get any updates (yes I know Solaris has that new Nightmare holds, but I've never seen them so until I see them ingame, I'm gonna assume they're not  merged yet).

**Comment by PhantomEpicness** [comment]
Ready for review

**Comment by neroisvcute** [comment]
Love seeing new additions to maps, thanks for making it.

**Comment by PhantomEpicness** [comment]
- Did nanu changes
- Added some lore papers
- Added additional story stuff

**Comment by PhantomEpicness** [comment]
Full narrative lore completed

**Review by Nanu308 — CHANGES_REQUESTED** [review]
Needs to be on the N wall, use Dir 1 and make sure it gets offset ingame onto the wall
bilde

This looks a bit weird with the wires, add another wire so it ends below them in a "wire knot" as well, ping me on Discord of confused.
bilde

Switch the floodlight crates out with a couple other things, as they can often times cause issues in their usage due to how certain machinery is handled as invulnerable.
bilde

File: maps/map_files/BigRed/BigRed.dmm [hunk 1]
```diff
@@ -1495,6 +1495,7 @@
 /obj/structure/pipes/standard/simple/hidden/green{
```
Comment (Nanu308) [inline maps/map_files/BigRed/BigRed.dmm:1495]: Reactor Insert needs a fix, offset into ETA is messy.

**Review by Nanu308 — CHANGES_REQUESTED** [review]

**Comment by Nanu308** [comment]
lgtm

**Pull request merged**